backoffice/views.py

```diff
@@ -1,12 +1,55 @@
 from django.shortcuts import render
 from django.contrib.auth.decorators import login_required
-from api.models import Animal, Entity
+import api.models as models
+from .forms import AnimalForm
+from django.utils import timezone
+from django.http import JsonResponse as json
 
 
 @login_required(login_url='login')
 def home(request):
+    from django.contrib.auth import authenticate
```

Comment:
Why is this import here? And where is it used?

Reply:
This is the remainings of an old bypass for the login method and to avoid a `@login_required`. It should (and will be) removed.